examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -311,6 +309,13 @@
     "\n",
     "plt.imshow(im_ego[::-1])"
    ]
+  },
+  {
```

Comment:
did you leave an empty cell at the end of the notebook?

Reply:
ah shoot...